### ISSUE TITLE
Local charts version fails validation (helm3 / helmsman beta4)

### ISSUE THREAD

**Issue by eperdeme** [Praqma/helmsman#362]
Hello,

I get the following error when trying to use beta4 and helm3.

I have a chart called 'thanos' which is local. The version of the chart is 0.4.0. This chart has a depdency which is prometheus-operator which is 8.5.0.

Helmsman seems to be findng the 8.5.0 version of the Charts depdency and failing, instead of the 0.4.0 version of the chart.

Am I doing something wrong here?

```
2020-01-02 12:38:52 INFO: Parsed YAML [[ helmsman.yaml ]] successfully and found [ 10 ] apps
2020-01-02 12:38:52 INFO: Validating desired state definition...
2020-01-02 12:38:52 INFO: Setting up kubectl...
2020-01-02 12:38:52 INFO: Setting up helm...
2020-01-02 12:39:01 INFO: Setting up namespaces...
2020-01-02 12:39:04 INFO: Validating charts...
2020-01-02 12:39:07 CRITICAL: Chart [ /Users/nflynn/git/trading_development/terraform_products/infrastructure/charts/thanos ] with version [ 0.4.0 ] is specified for app [thanos] but the chart found at that path has version [ 8.5.0 ] which does not match.
```

```
  thanos:
    namespace: prometheus
    enabled: true
    chart: ../charts/thanos
    version: 0.4.0
    priority: -99
    valuesFiles:
      - "apps/thanos.yaml"
```

```
➜  thanos git:(develop) ✗ cat Chart.yaml 
apiVersion: v2
appVersion: "1.0"
description: An opinionated Helm chart for Thanos + Prometheus Operator 
name: thanos
version: 0.4.0
dependencies:
  - name: prometheus-operator
    repository: '@stable'
    version: 8.5.0
```

**Comment by eperdeme** [comment]
I'm not really any good with go, but a wild guess it could be;

```charts git:(develop) ✗ helm inspect chart thanos
apiVersion: v2
appVersion: "1.0"
dependencies:
- name: prometheus-operator
  repository: '@stable'
  version: 8.5.0
description: An opinionated Helm chart for Thanos + Prometheus Operator
name: thanos
version: 0.4.0
```

https://github.com/Praqma/helmsman/blob/d33ce903e2bec4e97f5cd7eeea82641d60970c85/internal/app/release.go#L179

Does it match the first instance of version: it finds which is the dependency, or is there some logic that checks "version:" is at the begging of the line and not indented ? ?

**Comment by luisdavim** [comment]
fixed via #363
